dune

```diff
@@ -0,0 +1 @@
+(vendored_dirs vendor)
```

Comment:
This isn't necessary -- when I develop in a vendored tree, I just put this stanza inside a dune file in the vendor directory. See for example https://github.com/avsm/eeww/blob/main/lib/dune.

Reply:
Very nice, thanks! This is done now.